src/Routing/Router.php

```diff
@@ -206,9 +206,12 @@ public static function getNamedExpressions(): array
      * @throws \Cake\Core\Exception\CakeException
      * @see \Cake\Routing\RouteBuilder::connect()
      * @see \Cake\Routing\Router::scope()
+     * @deprecated 4.3.0 Use the `RouterBuilder`'s `connect()` method instead.
```

Comment:
I consciously avoided using `RouterBuilder::connect()` in the messages so as to prevent the not so bright users trying to use it statically and then complain.

Reply:
I thought so. It's up you, it's just a little awkwardly phrased.

Reply:
Yeah I not thrilled with the wording either.